style.css

```diff
@@ -98,3 +103,241 @@ a {
 p a {
   color: #7f8cff;
 }
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 2em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.pic1 img {
+  width: 100%;
+}
+
+.title1 {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  padding: 0%;
+  margin-left: -5px;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin-top: 0;
+}
+
+.language {
+  font-family: Poppins, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.html,
+.css,
+.java {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 12px;
+  padding: 10px;
+}
+
+.card-btn {
+  text-align: left;
+  background-color: white;
+  color: #396df2;
+  width: fit-content;
+  border-radius: 5px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  padding: 10px;
+}
+
+.card-btn:hover {
+  background-color: #396df2;
+  color: white;
+}
+
```

Comment:
- [ ] Good job for adding a `hover` state for a button 👍👍 I noticed that you missed the `box-shadow` for the `hover` state. Also kindly consider applying the `active` button interaction as well 🙌📌
- [ ] I noticed that you are using `border-radius: 5px`. But in the design guidelines, it is `8px`. Kindly fix that for getting a design close to the Figma template as much as possible.

Your hover interaction | Given design
---|---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/5dafca79-4c5e-4e5b-a2bc-0b17eca62888) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/10774210-6f89-49a1-885b-bf6e447df29d)

_Kindly apply this for all buttons_